docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+volumes:
+  andela_socials_db_data: {}
+  andela_social_db_backup: {}
+
+services:
+  postgres:
+    container_name: postgres_db
+    build:
+      context: ./docker/database/
+      dockerfile: Dockerfile
+    ports:
+      - 5434:5432
+
+    volumes:
+      - andela_socials_db_data:/var/lib/postgresql/data
+      - andela_social_db_backup:/backups
+
+
+  andela_socials:
```

Comment:
The naming is off, perhaps web service?

Reply:
Is it just me, I can't seem to see the services shown on the pix in any of these files

Reply:
@andela-sjames  Thanks I feel web service make sense we'll make necessary changes and we'll update the image to reflect the requested changes.